src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -422,8 +427,10 @@ private String parseStringOption(CommandLine commandline, String option, String
             log.info("Test Parameters: {}", toString());
 
             final ScheduledExecutorService bgExecutor = Executors.newScheduledThreadPool(10);
+            ClientConfig clientConfig = ClientConfig.builder().controllerURI(new URI(controllerUri))
```

Comment:
What about if the Segment Store is configured with TLS and the Controller is not, or vice-versa? Should we expose this as separate options in the configuration of the client?

Reply:
That is an atypical configuration that can be difficult to understand and and is really for a setup where TLS terminates Kubernetes Ingress/reverse proxy for the Controller. It isn't relevant for Pravega benchmark, in my view.